speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/rppUtils/RppUtils.hpp

```diff
@@ -95,4 +106,35 @@ inline bool SetupSingletonsFolder()
     }
 
     return true;
+}
+
+/// @brief Start running a separate executable with no command-line arguments
+/// @param p_executablePath The path to the executable.
+///// Should either be an absolute path or relative to the current directory.
+///// Path must include file extension; no default extension is assumed.
+///// Path is assumed to refer to an existing executable file
+inline void StartExecutable(const std::string& p_executablePath) {
+    // If compiling for Windows, use CreateProcess to run an executable
+#ifdef OS_WINDOWS
+    LPSTR args = _strdup(""); // Create an empty string of arguments for process
+    STARTUPINFO startupInformation = {sizeof(startupInformation)}; // Create an empty STARTUPINFO
+    PROCESS_INFORMATION processInformation; // Allocate space for PROCESS_INFORMATION
+    // Start the process. Nullpointers correspond to default values for this method.
+    // Inherit handles is not necessary for our use case and is thus false.
+    CreateProcess(p_executablePath.c_str(),
+                  args,
+                  nullptr,
+                  nullptr,
+                  false,
+                  0,
+                  nullptr,
+                  nullptr,
+                  &startupInformation,
+                  &processInformation
+    );
+
+    // TODO: If compiling for Linux, use a different method of starting an executable
+#elif OS_LINUX
```

Comment:
Shouldn't this TODO be done to actually finish this task? or is linux low priotity for now? if it is because you cant test linux is it possible write a potentially working version and commenting that?